Packs/Core/ReleaseNotes/3_0_39.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Indicators detection
+
+- Fixed an issue in CoreIRApiModule by adding a flag to the authorization process.
+
+##### Investigation & Response
+
+- Fixed an issue in CoreIRApiModule by adding a flag to the authorization process.
```

Comment:
1. the change is relevant only for XPANSE so to avoid confusion, let's hide the RN for other marketplaces
2. `CoreIRApiModule` is something that customers are not aware of, see my suggestion
```suggestion
<~XPANSE>
#### Integrations

##### Indicators detection

- Fixed an issue where the integration commands failed due to a change in the API request process.

##### Investigation & Response

- Fixed an issue where the integration commands failed due to a change in the API request process.
</~XPANSE>
```